src/components/shared/mediaText.js

```diff
@@ -202,7 +203,11 @@ function MediaText (props) {
     }
     headingClass = classNames(headingClass, headingColor);
     textCol = classNames(textCol, textColBg, textColHeight, textColPadding, "text-break");
-    
+
+    if (imageAlignment !== "left") {
+        wrapperCol = wrapperCol + " flex-row-reverse"
```

Comment:
```suggestion
        wrapperCol = classNames(wrapperCol, "flex-row-reverse");
```

Reply:
needs the space - otherwise it gets appended to the word before if one is there.

Reply:
No, you don't need the space if you use the `classNames` function - see the code for `textCol` on line 204